20-packages.ks

```diff
@@ -87,61 +88,53 @@ tcpdump
 #
 # Some ideas from:
 #
-# https://github.com/weldr/lorax/blob/rhel7-branch/share/runtime-cleanup.tmpl
-
-# Red Hat Enteprise Linux subscription tool
--subscription-manager
-
-# Generic and wireless tools and firmware
--prelink
--setserial
--ed
--authconfig
--wireless-tools
--iwl7260-firmware
--iwl3160-firmware
--iwl6000g2b-firmware
--iwl6000g2a-firmware
--iwl5000-firmware
--iwl6050-firmware
--iwl2030-firmware
--iwl135-firmware
--iwl2000-firmware
--iwl105-firmware
--iwl1000-firmware
--iwl6000-firmware
--iwl100-firmware
--iwl5150-firmware
--iwl4965-firmware
--iwl3945-firmware
```

Comment:
The iwl firmwares are for wireless adapters:
```console
# dnf search firmware | grep iwl
Last metadata expiration check: 0:00:32 ago on Wed Oct 30 16:23:56 2024.
iwl100-firmware.noarch : Firmware for Intel(R) Wireless WiFi Link 100 Series Adapters
iwl1000-firmware.noarch : Firmware for Intel® PRO/Wireless 1000 B/G/N network adaptors
iwl105-firmware.noarch : Firmware for Intel(R) Centrino Wireless-N 105 Series Adapters
iwl135-firmware.noarch : Firmware for Intel(R) Centrino Wireless-N 135 Series Adapters
iwl2000-firmware.noarch : Firmware for Intel(R) Centrino Wireless-N 2000 Series Adapters
iwl2030-firmware.noarch : Firmware for Intel(R) Centrino Wireless-N 2030 Series Adapters
iwl3160-firmware.noarch : Firmware for Intel(R) Wireless WiFi Link 3160 Series Adapters
iwl5000-firmware.noarch : Firmware for Intel® PRO/Wireless 5000 A/G/N network adaptors
iwl5150-firmware.noarch : Firmware for Intel® PRO/Wireless 5150 A/G/N network adaptors
iwl6000g2a-firmware.noarch : Firmware for Intel(R) Wireless WiFi Link 6005 Series Adapters
iwl6000g2b-firmware.noarch : Firmware for Intel(R) Wireless WiFi Link 6030 Series Adapters
iwl6050-firmware.noarch : Firmware for Intel(R) Wireless WiFi Link 6050 Series Adapters
iwl7260-firmware.noarch : Firmware for Intel(R) Wireless WiFi Link 726x/8000/9000/AX200/AX201 Series Adapters
```

I don't think we support wireless connections so then it may help to reduce the image. I don't know if they're still installed by default though.

Reply:
We had some issues reported from users who use FDI to provision laptops for their employees. They complained about the lack of drivers and support for WIFI.

Reply:
I don't mind if it's a deliberate choice. I honestly don't know how discovery would work in a wireless environment, but that shouldn't hold it back.